TheAurorian/src/main/resources/assets/theaurorian/patchouli_books/the_aurorian_guide/zh_cn/entries/magicalitems/aurorianslimeboots.json

```
@@ -0,0 +1,17 @@
+{
+    "name": "极光粘液靴子",
+    "icon": "theaurorian:aurorianslimeboots",
+    "category": "3magicalitems",
+    "readbydefault": true,
+    "pages": [
+        {
+            "type": "spotlight",
+            "item": "theaurorian:aurorianslimeboots",
+            "text": "$(#008cff)极光粘液靴子$()能让你在下蹲时跳得更高！它同样也会减少你的摔落伤害。默认情况下高跳之间有 5 秒的冷却。"
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
            "text": "$(#008cff)极光粘液靴子$()！它同样也会减少你的摔落伤害。默认情况下高跳之间有 5 秒的冷却。"
```
删掉的这句话需要实测。

---

实际上确实是这样